package.json

```diff
@@ -9,7 +9,9 @@
   "contributors": [
     "Danny Kijkov <daniel.kijkov@strv.com>"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "@commitlint/config-conventional": "^8.0.0"
```

Comment:
Please remove this from the PR, see #29.

Reply:
Done in 6ef5df5!